frontend/src/pages/modelServing/ModelServingRoutes.tsx

```diff
@@ -11,11 +11,17 @@ const ModelServingRoutes: React.FC = () => {
 
   return (
     <ProjectsRoutes>
-      <Route path="/" element={<ModelServingContextProvider />}>
+      <Route
+        path="/:namespace?/*"
+        element={
+          <GlobalModelServingCoreLoader
+            getInvalidRedirectPath={(namespace) => `/modelServing/${namespace}`}
+          />
+        }
+      >
         <Route index element={<ModelServingGlobal />} />
-        <Route path="/:namespace?/*" element={<ModelServingGlobal />} />
         <Route
-          path="/metrics/:project/:inferenceService"
+          path="metrics/:inferenceService"
```

Comment:
Since we elevate the outer wrapper of the route to `/:namespace?/*`, we don't need the `:project` here, and update the corresponding param in `<ModelServingMetricsWrapper />`